chanzuckerberg/idseq-web
# Update list of phages

# Description

The updates the list of taxids that we define as phages following the update of the NCBI databases to 2019-09-17.

# Notes

The total number of phages increased to 2726 from 2108.

# Tests

* deploy to staging
* open completed benchmark sample
* Try clicking "phages" in report filter 
* see no error
* see similar results before and after
